src/portal/azext_portal/generated/commands.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    with self.command_group('stream-analytics', is_experimental=True):
```

Comment:
command group name is not right.

Reply:
Since this pr already merged, I will address it in another pr. Thanks for your catch!